### ISSUE TITLE
Drag and Drop xblock not working properly inside the Mobile apps' webview 

### ISSUE THREAD

**Issue by miankhalid** [openedx/xblock-drag-and-drop-v2#375]
On long pressing to drag and drop an item:
1. the text on the item gets selected in case of iOS
2. and incase of Android the drag and drop doesn't work altogether

Attaching videos for reference ⬇️ 

| iOS | Android |
| - | - |
| <video src="https://github.com/openedx/xblock-drag-and-drop-v2/assets/1710804/a8d4a6ae-6f58-4060-8a8e-a577ccdca432"> | <video src="https://github.com/openedx/xblock-drag-and-drop-v2/assets/1710804/2be76a24-feb0-431a-9fdc-a1963e1179ed"> |

cc @forgotvas @moiz994 @touchapp @hassan6190

**Comment by miankhalid** [comment]
Hi @Agrendalath, this issue is affecting our (edX/2U's) prod mobile apps, can you help in fixing this issue? thanks!

**Comment by Agrendalath** [comment]
@miankhalid, to have the full context in this GitHub issue - is it related to the old or new mobile apps? I haven't worked with any of them, but I can ask if anybody has free CC hours and experience with relevant repos.

**Comment by miankhalid** [comment]
@Agrendalath this issue exists in the new apps as well, reason being both the new & old apps load the d&d xblock inside a webview.

Has there recently been a change in this xblock's working? cuz in the past it used to work just fine within the webviews of the mobile apps.

Behavior of the xblock in the new app on iOS ⬇️ 
<video src="https://github.com/openedx/xblock-drag-and-drop-v2/assets/1710804/55947f1d-4116-4e4e-aa71-d925bb23e032">

**Comment by Agrendalath** [comment]
@miankhalid, no, there were no direct changes to the dragging mechanism recently ([full history of the JS file](https://github.com/openedx/xblock-drag-and-drop-v2/commits/376ec5c/drag_and_drop_v2/public/js/drag_and_drop.js)).

If you could estimate when it stopped working and verify if [an older version of this XBlock](https://github.com/openedx/xblock-drag-and-drop-v2/releases/) works correctly, it would be really helpful, as this requires the mobile app setup process.
Just a quick note - releases newer than Palm will not work without [this commit](https://github.com/openedx/xblock-drag-and-drop-v2/commit/0b0bb0c80337e369a5c645915ef99056baa703b2). [This one](https://github.com/openedx/xblock-drag-and-drop-v2/pull/308/commits/db6130aeaa8e0b909b365f0fd290ec1183caccdb) may also be needed.

**Comment by miankhalid** [comment]
thanks! lemme get some help from my backend folks on this and get back to you, as I don't know how to configure an older version of this xblock on a sandbox.
